src/components/GoodsList/GoodsList.jsx

```diff
@@ -0,0 +1,14 @@
+import { GoodCard } from '../GoodCard/GoodCard';
+
+export const GoodList = ({ goods }) => (
+  <ul>
+    {
+      goods.map(good => (
+        <GoodCard
+          good={good}
+          key={goods.indexOf(good)}
```

Comment:
it's bad practice to use index as a key, you can use good as a key, or you can integrate `uuidv4` lib for making unique IDs

Reply:
Vitaliy. hi! Thanks for the review and your recommendations! I have fixed it.